packages/terra-alert/src/terra-dev-site/doc/alert/Alert.1.doc.mdx

```diff
@@ -1,3 +1,23 @@
+import { Badge } from 'terra-alert/package.json?dev-site-package';
+
+import AlertExample from '../example/AlertExample?dev-site-example';
+import ErrorExample from '../example/ErrorExample?dev-site-example';
+import WarningExample from '../example/WarningExample?dev-site-example';
+import AdvisoryExample from '../example/AdvisoryExample?dev-site-example';
+import InfoExample from '../example/InfoExample?dev-site-example';
+import SuccessExample from '../example/SuccessExample?dev-site-example';
+import CustomExample from '../example/CustomExample?dev-site-example';
+import LongTextExample from '../example/LongTextExample?dev-site-example';
+import ActionExample from '../example/ActionExample?dev-site-example';
+import DismissibleExample from '../example/DismissibleExample?dev-site-example';
+import ActionAndDismissibleExample from '../example/ActionAndDismissibleExample?dev-site-example';
+
+import AlertPropsTable from '../../../../src/Alert?dev-site-props-table';
+
+import ActionAndDismissableExampleDocs from '../../../../docs/action-and-dismissable-example.md';
```

Comment:
Is this import used anywhere?

Reply:
removed it here Updated here https://github.com/cerner/terra-core/pull/2813/commits/0ec3e9b2b11243ff5a0c25908ee356a9de91d5a8.